csfieldguide/static/interactives/city-trip/js/city-trip.js

```diff
@@ -0,0 +1,642 @@
+const cytoscape = require('cytoscape');
+const noOverlap = require('cytoscape-no-overlap');
+const automove = require('cytoscape-automove');
+const Mathjs = require('mathjs');
+
+cytoscape.use(noOverlap);
+cytoscape.use(automove);
+var stopPathFinding = false;
+var stoppedMidExecution = false; // true if the stop button been clicked mid path finding.
+
+const STATUS_COMPLETE = gettext("complete!");
+const STATUS_READY = gettext("ready to go!");
+const STATUS_STOPPED = gettext("stopped");
+const STATUS_RUNNING = gettext("running");
+
+$(document).ready(function() {
+
+  updateStatus(STATUS_READY, 'status-ready');
```

Comment:
Both maps will update, but they will be different if if the 'Add city' button is pressed after the algorithm completes
![image](https://user-images.githubusercontent.com/25916475/56101167-5433bd00-5f75-11e9-8f86-33a7fba518f3.png)

Reply:
Nice find!